PackageExplorer/Models/NuGetFrameworkModel.cs

```diff
@@ -0,0 +1,255 @@
+			frameWork.Items.Add(new NuGetTargetFrameworkItem { Name = ".NET Framework 4.5 and higher", ShortName = "net45", IsDefault = true, Framework = Frameworks.FindByIdentifierShortName("net45").FirstOrDefault() });
+			frameWork.SelectedItem = frameWork.Items.FirstOrDefault(x => x.IsDefault);
+			PortableFrameworks.Add(frameWork);
+
+			frameWork = new NuGetTargetFrameworkItem { Name = "Silverlight" };
+			frameWork.Items.Add(new NuGetTargetFrameworkItem { Name = "Silverlight 4 and higher", ShortName = "sl4", Framework = Frameworks.FindByIdentifierShortName("sl4").FirstOrDefault() });
+			frameWork.Items.Add(new NuGetTargetFrameworkItem { Name = "Silverlight 5", ShortName = "sl5", IsDefault = true, Framework = Frameworks.FindByIdentifierShortName("sl5").FirstOrDefault() });
+			frameWork.SelectedItem = frameWork.Items.FirstOrDefault(x => x.IsDefault);
+			PortableFrameworks.Add(frameWork);
+
+			frameWork = new NuGetTargetFrameworkItem { Name = "Windows Phone" };
+			frameWork.Items.Add(new NuGetTargetFrameworkItem { Name = "Windows Phone 7 and higher", ShortName = "wp7", Framework = Frameworks.FindByIdentifierShortName("wp7").FirstOrDefault() });
+			frameWork.Items.Add(new NuGetTargetFrameworkItem { Name = "Windows Phone 7.5 and higher", ShortName = "wp71", Framework = Frameworks.FindByIdentifierShortName("wp75").FirstOrDefault() });
+			frameWork.Items.Add(new NuGetTargetFrameworkItem { Name = "Windows Phone 8 and higher", ShortName = "wp8", Framework = Frameworks.FindByIdentifierShortName("wp8").FirstOrDefault() });
+			frameWork.Items.Add(new NuGetTargetFrameworkItem { Name = "Windows Phone 8.1 and higher", ShortName = "wpa81", IsDefault = true, Framework = Frameworks.FindByIdentifierShortName("wpa81").FirstOrDefault() });
+			frameWork.SelectedItem = frameWork.Items.FirstOrDefault(x => x.IsDefault);
+			PortableFrameworks.Add(frameWork);
+
+			frameWork = new NuGetTargetFrameworkItem { Name = ".NET for Windows Store apps", ShortName = "win8", Framework = Frameworks.FindByIdentifierShortName("win8").FirstOrDefault() };
+			PortableFrameworks.Add(frameWork);
+
+			frameWork = new NuGetTargetFrameworkItem { Name = "Xamarin For iOS", ShortName = "xamarinios", Framework = Frameworks.FindByIdentifierShortName("xamarinios").FirstOrDefault() };
+			PortableFrameworks.Add(frameWork);
+
+			frameWork = new NuGetTargetFrameworkItem { Name = "Xamarin For Android", ShortName = "MonoAndroid", Framework = Frameworks.FindByIdentifierShortName("MonoAndroid").FirstOrDefault() };
+			PortableFrameworks.Add(frameWork);
+		}
+	}
+
+	/// <summary>
+	/// Class NuGetTargetFrameworkItem.
+	/// </summary>
+	[PropertyChanged.ImplementPropertyChanged]
+	public class NuGetTargetFrameworkItem
+	{
+		/// <summary>
+		/// Gets or sets the name.
+		/// </summary>
+		/// <value>The name.</value>
+		public string Name { get; set; }
+
+		/// <summary>
+		/// Gets or sets the short name.
```

Comment:
Thanks for all the docs, but this won't help much. What is a short name? (maybe add an example?)